### PR TITLE
feat(ui): show active queue jobs and history in one list

### PR DESCRIPTION
**Pull request by hoivikaj**
## Summary
- Combines the Queue and History cards into a single list: active jobs stay at the top, and finished jobs remain in the same table as history.
- One search/filter/sort bar and one pager. Queue actions (pause, reorder, clear queue) and history actions (retry, clear failed/history) still apply to the matching rows.
- Dev preview: open `/queue?preview=1` for sample jobs.

## Test plan
- [ ] Open Queue and confirm there is a single card with active rows above a History divider.
- [ ] Complete or fail a job and confirm it leaves the active section and appears under History without a second card.
- [ ] Search, category, and status filters apply across the combined list (Downloading/Queued/Paused vs Completed/Failed).
- [ ] Select mixed rows and confirm delete/retry/pause only affect the matching item type.
- [ ] Open `/queue?preview=1` in local dev to review the populated layout.

## Summary by CodeRabbit

* **New Features**
  * Combined active jobs and completed history into one queue view.
  * Added shared search, filtering, sorting, pagination, selection, removal, retry, and history-clearing workflows.
  * Added grouped history rows, counts, status-specific actions, and bulk history retry support.
  * Added development-only preview data and an indicator when sample data is displayed.

* **UI Improvements**
  * Improved empty-state messaging, tooltip sizing, and completed/failed status badges.

* **Tests**
  * Added coverage for combined pagination, status filters, retry workflows, and queue/history edge cases.

### CODE REVIEW

File: frontend/app/routes/queue/combined-window.test.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  it("scopes active statuses to the queue", () => {
+    expect(statusAppliesToQueue("Downloading")).toBe(true);
+    expect(statusAppliesToHistory("Downloading")).toBe(false);
+  });
+
+  it("scopes finished statuses to history", () => {
+    expect(statusAppliesToQueue("Completed")).toBe(false);
+    expect(statusAppliesToHistory("Failed")).toBe(true);
+  });
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/combined-window.test.ts:69]: Added a dedicated test that an unrecognized status is excluded from both queue and history.

File: frontend/app/routes/queue/components/queue-table/queue-table.tsx [hunk 2]
```diff
@@ -245,6 +317,35 @@ export function QueueTable({
     await postClearQueue(clearCategory);
   }, [clearCategory]);
 
+  const onBulkRetry = useCallback(async () => {
+    if (selectedRetryableIds.length === 0) return;
+    setBulkRetryError(null);
+    const result = await retryHistoryItems(selectedRetryableIds);
+    if (!result.ok) {
+      setBulkRetryError(result.failed[0]?.error ?? "Failed to retry history items.");
+    }
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/components/queue-table/queue-table.tsx]: Bulk retry now shows the first failure whenever `result.failed` is non-empty, and calls `onHistoryRemoved` for `result.succeeded` so retried rows leave the table immediately.

File: frontend/app/routes/queue/components/queue-table/queue-table.tsx [hunk 3]
```diff
@@ -245,6 +317,35 @@ export function QueueTable({
...
+  }, [selectedRetryableIds]);
+
+  const onConfirmClearFailed = useCallback(async (deleteCompletedFiles?: boolean) => {
+    setIsConfirmingClearFailed(false);
+    try {
+      const url = `/api?mode=history&name=delete&value=failed&del_completed_files=${deleteCompletedFiles ? 1 : 0}`;
+      await fetch(url, { method: "POST" });
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/components/queue-table/queue-table.tsx:336]: Leaving Clear failed / Clear history as global SAB deletes (`value=failed` / `value=all`).

InfiniDysk history delete does not accept search, category, or status filters — only UUIDs, `value=all`, or `value=failed`. The confirm copy already says all failed / all history items will be removed, matching the previous history table and `docs/features/sab-api.md`. Passing list filters on the URL would not actually scope deletion without a new backend API.

Filter-scoped history delete would be a separate SAB-subset feature.

File: frontend/app/routes/queue/route.test.ts [hunk 4]
```diff
@@ -94,20 +90,17 @@ describe("queue route loader", () => {
     });
     expect(getConfigMock).toHaveBeenCalledWith(["api.categories", "api.manual-category"]);
     expect(result).toEqual({
-      queueSlots: fetchedQueueSlots.slice(1),
+      queueSlots: fetchedQueueSlots.slice(1, 6),
       previousQueueSlot: fetchedQueueSlots[0],
       nextQueueSlot: undefined,
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.test.ts]: The fixture now fills the  window plus a trailing sentinel, and the assertion names  /  as previous/next neighbors.

File: frontend/app/routes/queue/route.test.ts [hunk 5]
```diff
@@ -141,23 +134,17 @@ describe("queue route loader", () => {
...
-      historyPageSize: 100,
+      page: 1,
+      pageSize: 100,
     });
   });
 
-  it("passes independent queue and history filters to the server", async () => {
+  it("passes unified list filters to queue and history", async () => {
     getQueueMock.mockResolvedValueOnce({ slots: [], noofslots: 0 });
     getHistoryMock.mockResolvedValueOnce({ slots: [], noofslots: 0 });
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.test.ts]: Renamed the test to match the Paused-status routing, and dropped the unused history stub.

File: frontend/app/routes/queue/route.tsx [hunk 6]
```diff
@@ -1,7 +1,6 @@
 import { useRevalidator, useSearchParams } from "react-router";
 import type { Route } from "./+types/route";
-import { backendClient, type HistorySlot, type QueueSlot } from "~/clients/backend-client.server";
-import { HistoryTable } from "./components/history-table/history-table";
+import { backendClient } from "~/clients/backend-client.server";
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.tsx]: Imported  and  from the backend client so the presentation types typecheck. This was also the source of the frontend CI lint cascade.

File: frontend/app/routes/queue/route.tsx [hunk 7]
```diff
@@ -279,8 +295,13 @@ export default function Queue(props: Route.ComponentProps) {
     <div className="flex min-h-full min-w-full flex-col gap-8 px-4 py-4 text-sm text-base-content/70 md:px-8">
       <PageHeader
         title="Queue"
-        subtitle="Jobs from Sonarr, Radarr, or a manual NZB upload. Completed items move to History."
+        subtitle="Jobs from Sonarr, Radarr, or a manual NZB upload. Active items stay at the top; finished jobs remain in this list as history."
       />
+      {searchParams.get("preview") === "1" && (
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.tsx]: The preview banner now uses the same  guard as the loader, so production  cannot label live data as sample jobs.

File: frontend/app/components/ui/feedback.tsx [hunk 8]
```diff
@@ -98,7 +98,7 @@ export function Tooltip({
         role="tooltip"
         aria-hidden={!open}
         className={[
-          "tooltip-content z-50 w-72 max-w-[calc(100vw-2rem)] whitespace-normal text-left text-xs leading-relaxed",
+          "tooltip-content z-50 w-max max-w-[min(18rem,calc(100vw-2rem))] whitespace-normal text-left text-xs leading-relaxed",
```
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/components/ui/feedback.tsx]: Added  on tooltip content so long unbroken StatusBadge error strings wrap inside the width cap.

File: frontend/app/routes/queue/route.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.tsx]: Imported HistorySlot and QueueSlot from the backend client so the presentation types typecheck. This was also the source of the frontend CI lint cascade.

File: frontend/app/routes/queue/route.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.tsx]: The preview banner now uses the same import.meta.env.DEV guard as the loader, so production /queue?preview=1 cannot label live data as sample jobs.

File: frontend/app/components/ui/feedback.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/components/ui/feedback.tsx]: Added break-words on tooltip content so long unbroken StatusBadge error strings wrap inside the width cap.

File: frontend/app/routes/queue/route.test.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (hoivikaj) [reply frontend/app/routes/queue/route.test.ts]: The fixture now fills the queueLimit window plus a trailing sentinel, and the assertion names queue-before / queue-after as previous/next neighbors.
